assets/scripts/pages/admin/post.js

```diff
@@ -0,0 +1,71 @@
+import $ from "jquery";
+import 'datatables.net-bs5';
+import ajaxModal from '../../components/modal/ajax_modal';
+
+$(document).ready(function () {
+
+    $(".js-alert").fadeTo(4000, 0, function () {
+        $(".js-alert").addClass('d-none');
+    })
+
+    // Table des articles
+    $('#post-table').DataTable({
+        "order": [[1, "desc"]],
+        "keys": true,
+        "columnDefs": [
+        {
+            "targets": 4,
+            "sortable": false
+        },
+        ],
+        "language": {
+            "sEmptyTable": "Aucune donnée disponible dans le tableau",
+            "sInfo": "Affichage de l'élément _START_ à _END_ sur _TOTAL_ éléments",
+            "sInfoEmpty": "Affichage de l'élément 0 à 0 sur 0 élément",
+            "sInfoFiltered": "(filtré à partir de _MAX_ éléments au total)",
+            "sInfoPostFix": "",
+            "sInfoThousands": ",",
+            "sLengthMenu": "Afficher _MENU_ éléments",
+            "sLoadingRecords": "Chargement...",
+            "sProcessing": "Traitement...",
+            "sSearch": "Rechercher :",
+            "sZeroRecords": "Aucun élément correspondant trouvé",
+            "oPaginate": {
+                "sFirst": "Premier",
+                "sLast": "Dernier",
+                "sNext": "Suivant",
+                "sPrevious": "Précédent"
+            },
+            "oAria": {
+                "sSortAscending": ": activer pour trier la colonne par ordre croissant",
+                "sSortDescending": ": activer pour trier la colonne par ordre décroissant"
+            },
+            "select": {
+                "rows": {
+                    "_": "%d lignes sélectionnées",
+                    "0": "Aucune ligne sélectionnée",
+                    "1": "1 ligne sélectionnée"
+                }
+            }
+        }
+    });
+
+    $('#post-table tbody').on('click', '.js-post-delete', function () {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/magali-thuaire/blog-magali/pullRequest?prid=8924174)
```suggestion
    $("#post-table tbody").on('click', '.js-post-delete', function () {
```